### PR TITLE
Tweak docs

### PR DESCRIPTION
**Pull request by taldcroft**
@AustereCuriosity - once you review and merge then I can merge to astropy/master.

### CODE REVIEW

File: docs/io/fits/usage/table.rst [hunk 1]
```diff
@@ -371,9 +371,14 @@ process won't be necessary.
 FITS Table with Time Columns
 ============================
 
-The FITS standard defines the formats and keywords used to write timing information
-to FITS files and in many cases Astropy can just write a `~astropy.table.Table`
-with `~astropy.time.Time` columns in this format.
+The `FITS Time standard paper
+<http://adsabs.harvard.edu/abs/2015A%26A...574A..36R/>`_ defines the formats
+and keywords used to represent timing information in FITS files.  The astropy
```
Comment (aaryapatil) [inline docs/io/fits/usage/table.rst:376]: Should astropy be Astropy? Minor detail. But I think it should be kept same everywhere.

File: docs/io/unified.rst [hunk 2]
```diff
@@ -230,167 +230,177 @@ At this time, the ``meta`` attribute of the :class:`~astropy.table.Table` class
...
+    float64
+    --------
+         1.0
+         2.0
+
+Time
+~~~~
+
+By default, a `~astropy.time.Time` mixin column within `~astropy.table.Table`
+or `~astropy.table.QTable` will be written to FITS as a standard column using
```
Comment (aaryapatil) [inline docs/io/unified.rst:282]: within ``a`` `~astropy.table.Table`
Extremely minor change.

Also should ``column`` be ``Column``?

File: docs/io/unified.rst [hunk 3]
```diff
@@ -230,167 +230,177 @@ At this time, the ``meta`` attribute of the :class:`~astropy.table.Table` class
...
+the ``format`` of the object (similar to the output when printing to the
+console).  For instance if the ``format`` is ``iso`` then the FITS column with
```
Comment (aaryapatil) [inline docs/io/unified.rst:284]: FITS column ``will`` be a character...

File: docs/io/unified.rst [hunk 4]
```diff
@@ -230,167 +230,177 @@ At this time, the ``meta`` attribute of the :class:`~astropy.table.Table` class
...
+be a character string column.  None of the special header keywords associated
+with the Time FITS standard will be set.  When reading this back into Astropy
+the column will be an ordinary column instead of a `~astropy.time.Time` object.
+See the `Details`_ section below for an example.
+
+However, if the ``astropy_native`` keyword is ``True``, then the column
+will be written using the FITS Time standard by setting the necessary FITS
+header keywords.  Likewise with ``astropy_native=True`` when reading the
+table, any columns that conform to the FITS Time standard will be
+returned in the table as native `~astropy.time.Time` objects.  This allows
```
Comment (aaryapatil) [inline docs/io/unified.rst:294]: Should ``table`` be ``Table`` since we mean astropy Table here?

File: docs/io/unified.rst [hunk 5]
```diff
@@ -230,167 +230,177 @@ At this time, the ``meta`` attribute of the :class:`~astropy.table.Table` class
...
+Time
+~~~~
+
+By default, a `~astropy.time.Time` mixin column within `~astropy.table.Table`
+or `~astropy.table.QTable` will be written to FITS as a standard column using
+the ``format`` of the object (similar to the output when printing to the
+console).  For instance if the ``format`` is ``iso`` then the FITS column with
+be a character string column.  None of the special header keywords associated
+with the Time FITS standard will be set.  When reading this back into Astropy
+the column will be an ordinary column instead of a `~astropy.time.Time` object.
```
Comment (aaryapatil) [inline docs/io/unified.rst:287]: Same comment regarding ``column`` to ``Column``

**Review by aaryapatil — COMMENTED** [review]
This looks really good. I've left very minor comments. Should I merge this and do those changes myself?
(that is, if you approve those)

**Comment by taldcroft** [comment]
@AustereCuriosity - just merge and make the changes you noticed.

**Pull request merged**